### PR TITLE
8326110: [8u] The Marlin tests should be updated after JDK-8241307

### PR DESCRIPTION
**Pull request by mrserb**
The tests are updated to force the MarlinRenderingEngine.

@bourgesl please take a look.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] [JDK-8326110](https://bugs.openjdk.org/browse/JDK-8326110) needs maintainer approval

### Issue
 * [JDK-8326110](https://bugs.openjdk.org/browse/JDK-8326110): [8u] The Marlin tests should be updated after JDK-8241307 (**Bug** - P4 - Approved)

### Reviewers
 * [Laurent Bourgès](https://openjdk.org/census#lbourges) (@bourgesl - Committer) Review applies to [0ae5ee57](https://git.openjdk.org/jdk8u-dev/pull/447/files/0ae5ee57b7fb076c3c7fa14cbbf4f0fa2f6dc7a1)
 * [Paul Hohensee](https://openjdk.org/census#phh) (@phohensee - **Reviewer**)
 * [Andrew John Hughes](https://openjdk.org/census#andrew) (@gnu-andrew - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk8u-dev.git pull/447/head:pull/447` \
`$ git checkout pull/447`

Update a local copy of the PR: \
`$ git checkout pull/447` \
`$ git pull https://git.openjdk.org/jdk8u-dev.git pull/447/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 447`

View PR using the GUI difftool: \
`$ git pr show -t 447`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk8u-dev/pull/447.diff">https://git.openjdk.org/jdk8u-dev/pull/447.diff</a>

</details>
<details><summary>Using Webrev</summary>

[Link to Webrev Comment](https://git.openjdk.org/jdk8u-dev/pull/447#issuecomment-1950890713)
</details>

### CODE REVIEW

**Comment by bourgesl** [comment]
LGTM 😉

File: jdk/test/sun/java2d/marlin/CeilAndFloorTests.java
Comment (bourgesl) [inline jdk/test/sun/java2d/marlin/CeilAndFloorTests.java:1]: Not needed I think

File: jdk/test/sun/java2d/marlin/ArrayCacheSizeTest.java
Comment (bourgesl) [inline jdk/test/sun/java2d/marlin/ArrayCacheSizeTest.java:1]: Not needed I think

**Review by bourgesl — APPROVED** [review]
Maybe 2 unit tests do not need using the marlin engine to be enabled

File: jdk/test/sun/java2d/marlin/CeilAndFloorTests.java
Comment (mrserb) [reply jdk/test/sun/java2d/marlin/CeilAndFloorTests.java:1]: fixed.

File: jdk/test/sun/java2d/marlin/ArrayCacheSizeTest.java
Comment (mrserb) [reply jdk/test/sun/java2d/marlin/ArrayCacheSizeTest.java:1]: fixed.

**Comment by mrserb** [comment]
keep open

**Comment by mrserb** [comment]
keep open

**Comment by mrserb** [comment]
keep open

**Review by phohensee — APPROVED** [review]

**Review by gnu-andrew — APPROVED** [review]
Seems sensible to me.

**Comment by bourgesl** [comment]
Thanks, andrew

**Comment by gnu-andrew** [comment]
Moved to 8u452; see https://mail.openjdk.org/pipermail/jdk8u-dev/2024-December/019551.html

**Pull request closed without merging**